src/components/Tooltip.tsx

```diff
@@ -17,17 +17,20 @@ export default function Tooltip<T extends HTMLElement>({
   const [isHovered, setHovered] = useState(false);
 
   useEffect(() => {
-    if (!anchorRef?.current) {
-      return;
+    const element = anchorRef?.current;
+    if (!element) return;
+
+    function handleMouseEnter(this: HTMLElement) {
+      const rect = this.getBoundingClientRect();
```

Comment:
now compute rec client when user hovers

Reply:
@ap-justin @SovereignAndrey there's this weird edge case behavior that remained - if a user hovers the icon and immediately scrolls down/up, the tooltip follows the page scroll (see video below).

Not sure if this is a big deal, but it's there.

https://www.loom.com/share/3b4526712e064f09aab2d7cb715fe5cd?sid=d9eca95c-c267-4e61-8e9c-ccdb7c6a32e5